added a first pass of javascript. Monkeypatch code looks right to me, but maybe I messed it up. Either way, `+scoinJS/test` failing with:
```
X scoin.CryptoTest.schnorr - first schnorr signature test 8ms 
  utest.AssertionError: "E907831F80848D1069A5371B402410364BDF1C5F8307B0084C55F1CE2DCA821525F66A4A85EA8B71E482A74F382D2CE5EBEEE8FDB2172F477DF4900D310536C0" == sig.toHex.toUpperCase()
  sig: scoin.ByteVector64 = f3db32d4ee1dde4a034bfb0cca61640501da5ea88201491f1ed550883598950f99466af8fb1088971df5eb73197aa61911f06c86d42246e4a6fb92c4b679585f
    utest.asserts.Asserts$.assertImpl(main.js:23327)
    <jscode>.{anonymous}()(main.js:32423)
    scala.scalajs.runtime.AnonFunction0.apply(main.js:27808)
```

(note: the jvm test passes just fine)

--- a/js/src/main/scala/Bindings.scala
+++ b/js/src/main/scala/Bindings.scala
@@ -94,7 +94,7 @@ object monkeyPatch {
   }
 
   trait Sha256SyncFunctionType extends js.Function {
-    def apply(msg: Uint8Array):Uint8Array
+    def apply(msgs: Uint8Array*):Uint8Array
   }
 
   def init(): Unit = {
@@ -103,11 +103,8 @@ object monkeyPatch {
       hmacSha256Sync(key, msgs)
     }: HmacSha256SyncFunctionType)
 
-    Secp256k1.utils.asInstanceOf[js.Dynamic].sha256Sync = ({ (msg) =>
-      ByteVector.fromValidHex(
-        HashJS.sha256()
-        .update(ByteVector.view(msg).toHex,"hex")
-        .digest("hex")).toUint8Array
+    Secp256k1.utils.asInstanceOf[js.Dynamic].sha256Sync = ({ (msgs) =>
+      Crypto.sha256(ByteVector.concat(msgs.map(ByteVector.view(_)))).toUint8Array
     }: Sha256SyncFunctionType)
   }
 }

--- a/native/src/main/scala/CryptoPlatform.scala
+++ b/native/src/main/scala/CryptoPlatform.scala
@@ -226,6 +226,14 @@ private[scoin] trait CryptoPlatform {
       )
     )
 
+  def signSchnorrImpl(data: ByteVector32, privateKey: PrivateKey, auxrand32: Option[ByteVector32]): ByteVector64 = {
+    throw new NotImplementedError("implementation missing - signSchnorr - update sn-secp256k1 library and then come back and implement this")
+  }
+
+  def verifySignatureSchnorrImpl(data: ByteVector32, signature: ByteVector, publicKey: XonlyPublicKey): Boolean = {
+    throw new NotImplementedError("implementation missing - verifySignatureSchnorr - update sn-secp256k1 library and then come back and implement this")
+  }
+
   /** Recover public keys from a signature and the message that was signed. This
     * method will return 2 public keys, and the signature can be verified with
     * both, but only one of them matches that private key that was used to

